### PR TITLE
[10] Create & Style the Customer Signup Page  using CSS

### PR DESCRIPTION
**Pull request by MayowaShoyinka**
## Task: `[10] Create & Style the Customer Signup Page  using CSS`
- `rearranged my files to the right folder and deleted the not needed files`
signup-phone-view
signup-phone-view2
signup-tablet-view
signup-tablet-view2
signup-web-view

`

### CODE REVIEW

**Comment by MayowaShoyinka** [comment]
Changes Made @adaugochi

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  .footer-center input{ 
+    height:40px;
+}
+  .footer-container p{
+    text-align:center;
+    margin-top:20px;
+    padding-bottom:30px;
+  }
+}
+
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/css/signupstyle.css:211]: Remove spaces from line 211 - 214

**Review by adaugochi — COMMENTED** [review]
Well done.

Please create an `images` folder inside your `html` folder and place all your images there

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+}
+body{ 
+  margin: 0;
+  font-family:'Noto Serif', serif;
+  background-color:#faf6f2;
+  line-height:1.5;
+  height:100%;  
+}
+/*------------ Header Starts Here-------------*/
+.Navigation-header{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:12]: Update the class name to be in lowercase @MayowaShoyinka

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  overflow:hidden;
+  background-color:#116466;
+  padding:25px 4em 0 20px;
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:15]: Update the unit from `em` to `px`

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  line-height:50px;
+}
+.Navigation-header a{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:18]: Update the class name to be in lowercase @MayowaShoyinka

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+  float:left;
+  color:#fff;
+  text-align:center;
+  padding:20px;
+  text-decoration:none;
+  font-size:18px; 
+  line-height:25px;
+  border-radius:4px;
+}
+.Navigation-header a.logo{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:28]: Update the class name to be in lowercase @MayowaShoyinka

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+  font-size:25px;
+  font-weight:bold;
+}
+.Navigation-header-link{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:32]: Update the class name to be in lowercase @MayowaShoyinka

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+    font-size:15.2px;
+  }
+.footer-right img{
+  width:35px;
+   height:30px;
+}
+/* ------------ @ Media Queries (Responsiveness) ------------------------------- */
+@media screen and (max-width: 650px){
+  /*For header */
+.Navigation-header a{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:126]: Update the class name to be in lowercase @MayowaShoyinka

File: submissions/mayowaShoyinka/css/signupstyle.css [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+    float:none;
+    display:block;
+    text-align:left;
+    padding:4px;
+  } 
+  .Navigation-header-link{
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/css/signupstyle.css:132]: Update the class name to be in lowercase @MayowaShoyinka

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by MayowaShoyinka** [comment]
changes made @ToluAkin

**Pull request closed without merging**